src/homework-01/basic_types.ts

```diff
@@ -1,116 +1,120 @@
 class School {
-  directions = [];
+  directions: Direction[] = [];
 
-  addDirection(direction) {
+  addDirection(direction: Direction): void {
     this.directions.push(direction);
   }
 }
 
 class Direction {
-  levels = [];
+  levels: Level[] = [];
+  _name: string;
 
-  get name() {
+  get name(): string {
     return this._name;
   }
 
-  constructor(name) {
+  constructor(name: string) {
     this._name = name;
   }
 
-  addLevel(level) {
+  addLevel(level: Level) {
     this.levels.push(level);
   }
 }
 
 class Level {
-  groups = [];
+  groups: Group[] = [];
+  _name: string;
+  _program: any;
 
-  constructor(name, program) {
-    this.name = name;
+  constructor(name: string, program: any) {
+    this._name = name;
     this._program = program;
   }
 
-  get name() {
+  get name(): string {
     return this._name;
   }
 
-  get program() {
+  get program(): any {
     return this._program;
   }
 
-  addGroup(group) {
+  addGroup(group: Group) {

```

Comment:
Please, do not forget to add explicitly type annotation for the function return type.